rrfs-test/validated_yamls/templates/obtype_config/atms_npp_qc_bc.yaml

```diff
@@ -723,8 +578,4 @@ cost function:
            - name: reject
 
 
-test:
-  reference filename: Data/testinput/rrfs-mpasjedi-atms_npp.ref
-  test output filename: ./rrfs-mpasjedi-atms_npp.out
-  float relative tolerance: 1.0e-3
-  float absolute tolerance: 1.0e-6
+
```

Comment:
@SamuelDegelia-NOAA Do we need to keep the reference section here?

Reply:
No, we should not keep the reference section for individual obs space yamls like this. The plan (that I am currently working on testing) is to combine these individual obs space yamls into one larger yaml for each ctest type (EnVar, GETKF, etc.). Then those larger yamls will have their own reference section.